src/PKSim.Core/Services/IndividualMoleculeFactory.cs

```diff
@@ -125,14 +125,15 @@ protected TMolecule CreateMolecule(string moleculeName)
          CreateMoleculeParameterIn(molecule, REFERENCE_CONCENTRATION, CoreConstants.DEFAULT_REFERENCE_CONCENTRATION_VALUE, MOLAR_CONCENTRATION);
```

Comment:
We need to read this from the DB. We are duplicating so much info it does not make sense